rockcraft/extensions/express.py

```diff
@@ -0,0 +1,121 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""An extension for the NodeJS based Javascript application extension."""
+from typing import Any, Dict, Tuple
+
+from overrides import override
+
+from .extension import Extension
+
+
+class ExpressJSFramework(Extension):
+    """An extension for constructing Javascript applications based on the ExpressJS framework."""
+
+    @staticmethod
+    @override
+    def get_supported_bases() -> Tuple[str, ...]:
+        """Return supported bases."""
+        return "bare", "ubuntu@22.04", "ubuntu:22.04", "ubuntu@24.04", "ubuntu:24.04"
```

Comment:
I think we can just have the `@` notation. Could you please check which version of node comes with 22.04 and how commonly it is used? Then we can decide what we support